Pull from prod

WesGilster opened the pull request.

jmkao commented:
Now contains the `fullRelease` task that should do everything all at once.

`gradle clean fullRelease` should work so long as your repository can push to the desired release repo without conflicts.

jmkao commented:
@WesGilster How did `fullRelease` go for you? I'm seeing something weird on your releases page that I can't duplicate in my account, where you have two tags (DEV014 and DEV015) that point to the same commit, but have different release content. Did something strange happen when you tried using `fullRelease`?

WesGilster commented:
Yeah it did, my first one worked fine, but I forgot to commit anything to my local repo. I'm calling it a problem with my internet connection since I'm doing this all from rural Iowa, where the farms aren't know for it's fast speeds.

jmkao committed:
Merge pull request #272 from jmkao/master

Fix ZipImageFileProcessor

WGAndrew committed:
minor improvements

Make sure script runs as root
make sure script files are in unix format and runnable, even if they haven't been generated on a unix machine.
[originally changed in photocentric3d/photonic3d, but proposing for main branch]

WGAndrew committed:
Improved dos2unix calls

now greps for any shell scripts featuring carriage returns, builds a list, passes to xargs to pass to dos2unix

Means that it doesn't run dos2unix unless it absolutely has to.

WesGilster committed:
Merge pull request #276 from WGAndrew/patch-1

minor improvements to start.sh

WesGilster committed:
Merge pull request #278 from WesGilster/master

Fix for printing files without customizers

WesGilster committed:
Merge pull request #280 from WesGilster/master

Further fixes for Zip of images printing with scripting features

WesGilster committed:
Merge pull request #285 from WesGilster/master

Bugs fixes release

WesGilster committed:
Merge pull request #310 from WesGilster/master

Bug and functionality release
